orchestra/models/communication/managers.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class CommunicationPreferenceManager(models.Manager):
+
+    def get_or_create_all_types(self, worker, methods=None):
```

Comment:
interesting---get_or_create in the heading makes me think it will return them.  perhaps either return them, or call it `create_all_types_if_necessary`?